Lib/test/test_cmd_line.py

```diff
@@ -901,6 +901,11 @@ def res2int(res):
         )
         self.assertEqual(res2int(res), (6000, 6000))
 
+    def test_cpu_count(self):
+        code = "import os; print(os.cpu_count())"
+        res = assert_python_ok('-X', 'cpu_count=4321', '-c', code)
+        self.assertEqual(res.out.strip().decode("utf-8"), '4321')
```

Comment:
That's a lot of CPUs, it's exciting! Until *recently*, Linux was limited to 4096 CPUs! https://unix.stackexchange.com/questions/4507/how-many-cores-can-linux-kernel-handle